app/identifiers/register/establishers/company/CompanyAddressYearsId.scala

```diff
@@ -20,11 +20,20 @@ import identifiers.TypedIdentifier
 import identifiers.register.establishers.EstablishersId
 import models.AddressYears
 import play.api.libs.json.JsPath
+import utils.Cleanup
 
 case class CompanyAddressYearsId(index: Int) extends TypedIdentifier[AddressYears] {
   override def path: JsPath = EstablishersId.path \ index \ CompanyAddressYearsId.toString
 }
 
 object CompanyAddressYearsId {
   override lazy val toString: String = "companyAddressYears"
+
```

Comment:
We need to add some tests like [this](https://github.com/hmrc/pensions-scheme-frontend/blob/master/test/identifiers/register/establishers/individual/AddressYearsIdSpec.scala)